ca/src/main/resources/providers/va_facilities.csv

```diff
@@ -0,0 +1,3 @@
+id,name,address,city,state,zip,phone,type,ownership,emergency,quality,LAT,LON,EBTPU,PCT,PRRP,SIPU,PTSD_DOM,DH,WSDTT,SUPT,WTRP
+42,Matz Fake veteran hospital- sunnybrook health sciences centre,bayview ave,Toronto,on,416-687-2000,"VA Facility",Government,no,"Not Available",43.720661,-79.378057,,,,,,,,,
```

Comment:
You are missing a comma in the `va_facilities.csv` file between `STATE,ZIP,PHONE` columns, which is putting the LAT and LON in the wrong spaces.

You have `...Toronto,on,416-687-2000...` but it should be `...Toronto,on,,416-687-2000...`

Notice the extra comma after `on`.

Reply:
If you fix that comma it works fine on my machine. See if that works for you.